Jenkinsfile

```diff
@@ -155,7 +155,13 @@ pipeline {
         sh "./scripts/release.sh --alias-tag ci --registry ${env.REGISTRY}"
         withCredentials([file(credentialsId: 'kubeconfig', variable: 'KUBECONFIG')]) {
           sh 'kubectl get nodes -o wide'
-          sh "nix-shell --run './scripts/e2e-test.sh --device /dev/nvme1n1 --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\"'"
+          script {
+            def cmd = "./scripts/e2e-test.sh --device /dev/nvme1n1 --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\""
```

Comment:
Are you sure this will always be the device you think it is?

Reply:
Right. For now it is a safe choice. When we test multiple clouds and different devices all of this will need to get changed so we will figure that out when we get there 👍